src/components/users/LoginForm.js

```diff
@@ -16,37 +17,42 @@ const LoginForm = () => {
     const { email, password } = inputs;
     // cleanAlert is to remove error alert on change. This will be erased
     const [cleanAlert, setCleanAlert] = useState(false);
+    const [isSubmitted, setIsSubmitted] = useState(false);
     // function to update the inputs when the user modifies the input
-    function handleChange(e) {
+    const handleChange = ({ target }) => {
+        setIsSubmitted(false)
         if (!loggingIn) {
-            const { name, value } = e.target;
+            const { name, value } = target;
             setInputs(inputs => ({ ...inputs, [name]: value }));
             setCleanAlert(true);
         }
     }
     // get dispatch function of the Redux store
     const dispatch = useDispatch();
     // handle submit of the form
-    function handleSubmit(e) {
+    const handleSubmit = (e) => {
         e.preventDefault();
+        setIsSubmitted(true)
         if (email && password) {
             setCleanAlert(false);
             dispatch(userActions.login(email, password));
         }
     }
     // get flag to see if the user is logging in
-    const loggingIn = useSelector(state => state.authentication.loggingIn);  
+    const loggingIn = useSelector(state => state.authentication.loggingIn);
     return (      
         <form align="center" className="login-form" onSubmit={handleSubmit}>
             <p className="login-form__text">EMAIL</p>
             <input className="login-form__input" type="email" name="email" value={email} onChange={handleChange}/>
+            {isSubmitted && !email && <div className="login-form__alert"><FormattedMessage id="loginform.missing.email.text"/></div>}
             <p className="login-form__text">PASSWORD</p>
             <input className="login-form__input" type="password" name="password" value={password} onChange={handleChange} autoComplete="on"/>
+            {isSubmitted && !password && <div className="login-form__alert"><FormattedMessage id="loginform.missing.pass.text"/></div>}
             <div>
                 <button type="submit" className="login-form__btn-text"><FormattedMessage id="loginform.signin.text"/></button>
             </div>
             {loggingIn && <Loader type="ThreeDots" color="#2FBCF7" height={80} width={50}/>}
-            {!loggingIn && !(cleanAlert) && alert && alert.message && <div>{alert.message}</div>}
+            {isSubmitted && !loggingIn && !cleanAlert && alert && <div className="login-form__alert"> {alert.message} </div>}
```

Comment:
maybe you should move `isSubmitted && !loggingIn && !cleanAlert && alert` to a constant, `showLoginAlert`?